iOS Client/HTTPClient/MBClient.swift

```diff
@@ -52,6 +52,14 @@ class MBClient: NSObject {
             case .mockingCast: return "mockingcast"
             }
         }
+        
+        var title: String {
+            switch self {
+            case .pz: return "PZ Podcast"
+            case .mockingCast: return "The MockingCast"
+            case .mockingPulpit: return "Mocking Pulpit"
```

Comment:
Not completely sure, but I think we want `The Mockingpulpit` `The Mockingcast` and `PZ's Podcast` based on the titles that show up in iTunes (which seem to be derived from the Title field at the top of the XML returned by those 3 URLs . . . perhaps we can dynamically figure it out instead of hard-coding it . . . Not completely sure. Hard-coding isn't a major issue since the urls have the name as well, but It'd be stylish if we could rely on the data source)

Reply:
Yeah I'll change that to the strings for now and maybe if we get to it we can parse out the podcast name from the rss feed